src/main/java/com/google/sps/servlets/YoutubePost.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.servlets.YoutubeApiException;
+
+// Data model for a singular Youtube post that will be added into a list of posts.
```

Comment:
Javadoc is following the format:

```java
/** Data model for a singular Youtube post that will be added into a list of posts. */
```

Note the leading `/**` and trailing `*/`, which are used to indicate that the comment is part of the doc